extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/thing/ZonePlayer.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As the BRIDGE and the SUB cannot play audio by themselves, are considered as "invisible" and are hidden from the user interface of control points, we do not provide specific ThingTypes for them.